i18n/ja.json

```diff
@@ -301,27 +301,27 @@
   },
   {
     "id": "api.channel.post_update_channel_displayname_message_and_forget.create_post.error",
-    "translation": "表示名更新メッセージ %v を投稿できませんでした"
+    "translation": "表示名更新メッセージを投稿できませんでした"
   },
   {
     "id": "api.channel.post_update_channel_displayname_message_and_forget.retrieve_user.error",
-    "translation": "チャンネル表示名更新メッセージ%vを保存する際にユーザーを取得できませんでした"
+    "translation": "チャンネル表示名を更新する際にユーザーを取得できませんでした"
   },
   {
     "id": "api.channel.post_update_channel_displayname_message_and_forget.updated_from",
     "translation": "%sがチャンネル表示名を %s から %s に更新しました"
   },
   {
     "id": "api.channel.post_update_channel_header_message_and_forget.post.error",
-    "translation": "Failed to post update channel header message"
+    "translation": "チャンネルヘッダー更新メッセージを投稿できませんでした"
   },
   {
     "id": "api.channel.post_update_channel_header_message_and_forget.removed",
     "translation": "%sはチャンネルヘッダーを削除しました(これまでは%sでした)"
   },
   {
     "id": "api.channel.post_update_channel_header_message_and_forget.retrieve_user.error",
-    "translation": "チャンネルのヘッダー更新メッセージ%vを保存する際にユーザーを取得できませんでした"
+    "translation": "チャンネルのヘッダーメッセージ%vを更新する際にユーザーを取得できませんでした"
```

Comment:
This message has `%v` parameter. [en.json](https://github.com/mattermost/platform/blob/e9c9688b343049c6d461260bd15fff3486238f92/i18n/en.json#L324)
But no parameters are given [when the message is used](https://github.com/mattermost/platform/blob/d3a285e64d051aa8d5c4c9854597dfbcce107675/app/channel.go#L410)

`%v` should be removed.